keyboards/whale/sk/v3/info.json

```diff
@@ -0,0 +1,970 @@
+{
```

Comment:
This file doesnt contain valid content
1. extra layout macros that do not exist in v3.h header
2. DEFAULT should be LAYOUT_all
3. DEFAULT thumb cluster order is invalid (tested with "fast input" mode on configurator)

Reply:
I cannot find a way to validate this .json file, any suggestion?

Reply:
I have both v3.h & info.json updated.